src/client/editor/ConflictResolution.js

```diff
@@ -0,0 +1,97 @@
+export default class ConflictHandler {
```

Comment:
Generally, if a module exports a single default class, I think we want the file name to match the class name.